api/routes/cartRouter.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+
+const { validateToken } = require("../middleware/auth");
+
+const cartController = require("../controllers/cartController");
+
+const router = express.Router();
+
+router.get("/", validateToken, cartController.getUserCarts);
+
+router.post("/", validateToken, cartController.postCart);
+
+router.delete("/:cartId", cartController.deleteCart);
+
+module.exports = {
+  router,
+};
```

Comment:
아래 예시를 참고하여 간격을 수정해 주세요.
```suggestion
const express = require("express");
const cartController = require("../controllers/cartController");
const validation = require('../middlewares/auth');
const router = express.Router();

router.get("/list", validation.validateToken, cartController.getCartList);
router.post("/:productId", validation.validateToken, cartController.addProduct);
router.delete("/:cartId", validation.validateToken, cartController.deleteCart);
router.patch("/:cartId", validation.validateToken, cartController.updateAmount);

module.exports = {
    router
}```